example/src/main/scala/HttpsClient.scala

```diff
@@ -0,0 +1,42 @@
+import io.netty.handler.ssl.SslContextBuilder
```

Comment:
Wrap the netty SSL builder using a custom one.

Reply:
Discussed already. There are n api for creating `truststore` and `keystore` using SslContextBuilder.